packages/editor/src/components/block-compare/block-view.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
I'm not sure what you mean by editors - can you point me in the direction?

Reply:
Can't speak for Andrew but since he's asleep I'll take an educated guess. Maybe he meant `edit-post`?

<img width="407" alt="screen shot 2018-09-12 at 10 49 29" src="https://user-images.githubusercontent.com/1204802/45413479-98767c00-b679-11e8-93f0-615c470a10eb.png">

Reply:
Again I can't but make educated guesses — I think it's something about the packages getting published on NPM but not sure.

Reply:
I'll wait for Andrew to clarify, but the component is already in the `editor` package, not in the `components` package, so maybe it was a misread.

Reply:
Oof, I'm sorry for the trouble, it was definitely a misread on my part.

I had mistaken it as being part of `packages/components`.